packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -515,13 +506,13 @@ const convertEventToChatMessage = (
 const isChatMessageEditedEvent = (
   event: ChatMessageReceivedEvent | ChatMessageEditedEvent | ChatMessageDeletedEvent
 ): event is ChatMessageEditedEvent => {
-  return event['editedOn'] !== undefined;
+  return 'editedOn' in event && event['editedOn'] !== undefined;
```

Comment:
Oh I didn't know about the `in` check!
I believe this would actually be better as just the `in` check and remove the `event['editedOn'] !== undefined` (but also fine to leave as you have it)

Reply:
It probably would be, as it should amount to the same thing.

Reply:
Updatd.